File: cesta/AMarotoChivite-cesta/AMarotoChivite-cesta-Java/src/main/java/simulacionCesta/factories/FinalCestFactory.java
package simulacionCesta.factories;

import simulacionCesta.models.CestList;
import simulacionCesta.models.FinalCest;

import java.time.LocalDate;

public class FinalCestFactory {
    public static FinalCest create() {
        CestList cestList = CestListFactory.create();
        LocalDate dateCreation = LocalDate.now();
        return new FinalCest(dateCreation, cestList);
    }
}

File: cesta/AMarotoChivite-cesta/AMarotoChivite-cesta-Kotlin/src/main/kotlin/simulacionCesta/factories/FinalCestFactory.kt
package simulacionCesta.factories

import simulacionCesta.models.FinalCest
import java.time.LocalDate

object FinalCestFactory {
    @JvmStatic
    fun create(): FinalCest {
        val cestList = CestListFactory.create()
        val dateCreation = LocalDate.now()
        return FinalCest(dateCreation, cestList)
    }
}